components/StyledComponents/StyledButtonIcon.js

```diff
@@ -4,5 +4,26 @@ const StyledButtonIcon = styled.button`
   color: #666666;
   background-color: transparent;
   border: none;
+
+  &.button--deleteIcon {
+    background-color: transparent;
+    box-shadow: none;
+    margin-left: auto;
+  }
+
+  &.button--starIcon {
+    ${({ isActive }) => {
+      if (isActive) {
+        return `       
+        color: #ffd700;
+        padding: 0 0 0 1rem;
+    `;
+      } else {
+        return `
+        color: #eee3e8;
+        padding: 0 0 0 1rem;`;
+      }
+    }}
+  }
```

Comment:
```suggestion
  &.button--starIcon {
  color: #eee3e8;
  padding: 0 0 0 1rem;
    ${({ isActive }) => 
      isActive && `       
        color: #ffd700;
        padding: 0 0 0 1rem;
    `;
    }
  }
```

So könntest du es auch lösen, ist dir überlassen, was dir persönlich besser gefällt